backend/typescript/models/petCareInfo.model.ts

```diff
@@ -0,0 +1,13 @@
+import { Column, Model, Table } from "sequelize-typescript";
+
+@Table({ tableName: "pet_care_info" })
```

Comment:
Should there be a timestamp: false here?

Reply:
You're right, I'll add that rn